### PR TITLE
feat: Add error-focus-outline shadow

### PR DESCRIPTION
**Pull request by ynotdraw**
## 🚀 Description
This PR introduces a new error+focus style

---

## 🔬 How to Test

- Tests should pass
- View https://add-new-error-focus-outline.tailwind-toucan-base.pages.dev
- Scroll down to Shadows and verify you see the new shadow added

---

## 📸 Images/Videos of Functionality

Screenshot 2023-03-09 at 2 54 50 PM

### CODE REVIEW

File: tests/__snapshots__/css-import.test.ts.snap [hunk 1]
```diff
@@ -462,11 +462,6 @@ input::-moz-placeholder, textarea::-moz-placeholder {
   color: #a1a1aa;
 }
 
-input:-ms-input-placeholder, textarea:-ms-input-placeholder {
-  opacity: 1;
-  color: #a1a1aa;
-}
```
Comment (ynotdraw) [inline tests/__snapshots__/css-import.test.ts.snap:468]: @NullVoxPopuli do you happen to know why the prefixes would be stripped out here?  All I did was add a shadow 😭

Comment (NullVoxPopuli) [reply tests/__snapshots__/css-import.test.ts.snap:468]: my hunch would be autoprefixer / browserslist changes (which involve floating deps).

the `-ms` prefix was for IE, and idk if anyone uses IE anymore

Comment (ynotdraw) [reply tests/__snapshots__/css-import.test.ts.snap:468]: Niceeeee! I figured it was likely something with autoprefixer.  I think this is mostly safe.  There's some `-webkit` stuff in here too, but everything _looks_ safe to me.

**Review by simonihmig — APPROVED** [review]

**Review by nicolechung — APPROVED** [review]

**Comment by ynotdraw** [comment]
This failed release in CI - having convos about it now to get us unblocked.  It wants two users with write access as approvers, even though we all have write access 🙃.  Likely due to branch protection settings.

**Pull request merged**